bin/install-php-linter

```diff
@@ -0,0 +1,15 @@
+#!/bin/bash
+set -e
+
+cd bin
```

Comment:
The script is already inside the `bin` folder. Is this really necessary?

Reply:
yes, otherwise the folders are created outside of `bin`